src/lxc/tools/lxc_stop.c

```diff
@@ -155,13 +154,6 @@ int main(int argc, char *argv[])
 		exit(ret);
 	}
 
-	/* shortcut - if locking is bogus, we should be able to kill
-	 * containers at least */
```

Comment:
Do we have another approach to working around this?

Reply:
Yes, I thought about this. I think we can send SIGKILL to the monitor process itself which will also kill the container since we now correctly set the death signal.